firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -229,6 +232,63 @@ final class SearchSettingsTableViewController: ThemedTableViewController, Featur
 
         case .firefoxSuggestSettings:
             switch indexPath.item {
+            case FirefoxSuggestItem.browsingHistory.rawValue:
+                let setting = BoolSetting(
+                    prefs: profile.prefs,
+                    theme: themeManager.currentTheme,
+                    prefKey: PrefsKeys.SearchSettings.showFirefoxBrowsingHistorySuggestions,
+                    defaultValue: model.shouldShowBrowsingHistorySuggestions,
+                    titleText: String.localizedStringWithFormat(
+                        .Settings.Search.Suggest.SearchBrowsingHistory
+                    )
+                )
+                setting.onConfigureCell(cell, theme: themeManager.currentTheme)
+                setting.control.switchView.addTarget(
+                    self,
+                    action: #selector(didToggleBrowsingHistorySuggestions),
+                    for: .valueChanged
+                )
+                cell.editingAccessoryView = setting.control
+                cell.selectionStyle = .none
```

Comment:
nit: all these switch cases, with the one exception, look to be the same with minor differences is some passed in values. We could probably improve this by extracting into a `buildSettingWith(key: defaultValue: titleText: statusText: cell: selector: )`? It's not the end of the world if we don't, but worth mentioning.